src/placeos-models/user.cr

```diff
@@ -12,11 +12,37 @@ module PlaceOS::Model
   class User < ModelBase
     include RethinkORM::Timestamps
 
+    struct Email
+      getter address : String
+      delegate to_json, to: address
```

Comment:
maybe `forward_missing_to address` instead of `delegate`

Reply:
that's cool